ansible/README.md

```diff
@@ -0,0 +1,84 @@
+Deploying Openwhisk using Ansible
+=========
+
+
+### Getting started
+
+If you want to deploy Openwhisk locally using Ansible, you first need to install Ansible on your development environment:
+
+#### Ubuntu users
+```
+sudo apt-get install python-pip
+sudo pip install ansible==2.0.2.0
+```
+
+#### Vagrant users
+Nothing to be done, Ansible is already installed during vagrant provisioning.
+
+#### Mac users
+It is assumed that a VM has been provisioned using **docker-machine**.
+
+```
+brew install python
+pip install ansible==2.0.2.0
+
+cd ansible/environments/mac
+./genHosts.sh <name of docker-machine>
+```
+After this there should be a `hosts` file in the `ansible/environments/mac` dir.
+
+To verify the hosts file you can do a quick ping to the docker machine:
+```
+ansible all -i ansible/enviroments/mac -m ping
```

Comment:
note that this command is relative to openwhisk root but the previous `cd` has already changed working directory.

running this command produced the following error:
``
> ansible all -i ansible/enviroments/mac -m ping
ERROR! Unexpected Exception: (setuptools 1.1.6 (/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python), Requirement.parse('setuptools>=11.3'))
```